### ISSUE TITLE
[FEATURE REQUEST] shodan.io plugin

### ISSUE THREAD

**Issue by micrie** [jerlendds/osintbuddy#62]
It would be great to have a shodan.io plugin available.

The API key could be added on a special settings page in the Plugin section in the Settings menu.

**Comment by jerlendds** [comment]
I completely agree, I've been intending to add a Shodan plugin eventually. I'm working on some changes to the plugin system at the moment, once those are done the plugins will have access to a settings API which could be used for this
